app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> bool:
+    try:
+        command, file, new_file = command.split(" ")
+        if file != new_file and command == "cp":
+            with open(file) as file, open(new_file, "a+") as new_file:
+                new_file.write(file.read())
+        else:
+            return False
+    except (ValueError, FileNotFoundError, PermissionError):
+        return False
+    else:
+        return True
```

Comment:
why do u need this returns here? it seems to me, that this func shouldn't return smth

Reply:
In documentation for this function we don't have info about return or print smth. So I created this function like I wont